### PR TITLE
Move all Event client side React files from ./src/apps to ./src/client

### PR DESCRIPTION
**Pull request by vfarah-if**
## Description of change

Test all event works as it used to

## Test instructions

_What should I see?_

## Screenshots
image

_Add a screenshot_

image

_Add a screenshot_

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"

- [x] Has the branch been rebased to master?
- [x] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, Edge, Safari)

### CODE REVIEW

File: src/client/index.jsx [hunk 1]
```diff
@@ -42,15 +42,15 @@ import Dashboard from './components/Dashboard'
 import PersonalisedDashboard from './components/PersonalisedDashboard'
 import CompanyLocalHeader from './components/CompanyLocalHeader'
 import CompanyOrdersCollection from '../client/modules/Omis/CollectionList/CompanyOrdersCollection'
-import EventDetails from '../apps/events/client/Details/EventDetails.jsx'
+import EventDetails from './modules/Events/EventDetails/'
```
Comment (tomkingston) [inline src/client/index.jsx:45]: Shouldn't need a trailing slash

File: src/client/modules/Events/EventDetails/index.jsx [hunk 2]
```diff
@@ -6,18 +6,14 @@ import Button from '@govuk-react/button'
...
 import styled from 'styled-components'
 import urls from '../../../../lib/urls'
-import Task from '../../../../client/components/Task'
+import Task from '../../../components/Task'
 import { TASK_GET_EVENT_DETAILS, ID, state2props } from './state'
-import { EVENTS__DETAILS_LOADED } from '../../../../client/actions'
-import LocalHeader from '../../../../client/components/LocalHeader/LocalHeader.jsx'
-import { Main } from '../../../../client/components'
+import { EVENTS__DETAILS_LOADED } from '../../../actions'
+import LocalHeader from '../../../components/LocalHeader/LocalHeader.jsx'
```
Comment (tomkingston) [inline src/client/modules/Events/EventDetails/index.jsx:12]: Shouldn't need to specify `.jsx`. 
If `LocalHeader` is specified in components/index.js then you should be able to just `import { LocalHeader } from '../../../components'`, which you could combine with all the other imports belowbelow

File: src/client/modules/Events/EventDetails/index.jsx [hunk 3]
```diff
@@ -6,18 +6,14 @@ import Button from '@govuk-react/button'
...
+import { Main } from '../../../components'
 
-import {
-  SummaryTable,
-  FormActions,
-  NewWindowLink,
-} from '../../../../client/components'
-import { LocalNav, LocalNavLink } from '../../../../client/components'
+import { SummaryTable, FormActions, NewWindowLink } from '../../../components'
+import { LocalNav, LocalNavLink } from '../../../components'
```
Comment (tomkingston) [inline src/client/modules/Events/EventDetails/index.jsx:16]: You could import all of these in one go:
```
import {
  FormActions,
  LocalHeader,
  LocalNav,
  LocalNavLink,
  Main,
  NewWindowLink,
  SummaryTable,
} from '../../../components'
```

**Review by tomkingston — COMMENTED** [review]
A couple of minor comments, but looks good

File: src/client/modules/Events/EventDetails/index.jsx (hunk 2 above)
Comment (vfarah-if) [reply src/client/modules/Events/EventDetails/index.jsx]: Fixed

File: src/client/index.jsx (hunk 1 above)
Comment (vfarah-if) [reply src/client/index.jsx]: Fixed

File: src/client/modules/Events/EventForm/index.jsx [hunk 4]
```diff
@@ -1,13 +1,14 @@
...
+import { useParams } from 'react-router-dom'
 
 import GridRow from '@govuk-react/grid-row'
 import GridCol from '@govuk-react/grid-col'
 
 import urls from '../../../../lib/urls'
-import LocalHeader from '../../../../client/components/LocalHeader/LocalHeader.jsx'
-import { Main } from '../../../../client/components'
-import TaskForm from '../../../../client/components/Task/Form'
+import LocalHeader from '../../../components/LocalHeader/LocalHeader.jsx'
```
Comment (tomkingston) [inline src/client/modules/Events/EventForm/index.jsx:9]: Just `import { LocalHeader, Main } from '../../../components'`

**Review by chloehkwong1 — APPROVED** [review]

**Review by craigjbass — APPROVED** [review]

**Pull request merged**